lib/product-import.coffee

```diff
@@ -15,8 +16,10 @@ class ProductImport
     if options.blackList and ProductSync.actionGroups
       @sync.config @_configureSync(options.blackList)
     if options.ensureEnums then @ensureEnums = options.ensureEnums else @ensureEnums = false
+    if options.filterUnknownAttributes then  @filterUnknownAttributes = options.filterUnknownAttributes else @filterUnknownAttributes = false
```

Comment:
This should work `@filterUnknownAttributes = options.filterUnknownAttributes or false`